lib/types/types.d.ts

```diff
@@ -113,6 +113,10 @@ export type CallState = {
         [userID: string]: boolean;
     };
 };
+export type CallStateData = {
+    channel_id: string;
+    call: string;
```

Comment:
I'm crying inside. 😭

Reply:
funny we have two strongly typed languages both backend and frontend but have to resort to this :man_facepalming: